services/core-api/app/api/document_generation/resources/now_document.py

```diff
@@ -77,7 +81,14 @@ def get(self):
         now_application_identity.now_application.documents.append(now_doc)
         now_application_identity.save()
 
-        # Return the generated document
+        now_application = NOWApplication.find_by_application_guid(now_application_guid)
+        now_application_document_type.after_template_generated(template_data, now_doc,
+                                                               now_application)
+
+        # Depending on the return_record param, return the document record or file content
```

Comment:
Everywhere in our code that calls this endpoint currently is configured to receive the record rather than the file stream, so it's possible that we could just remove this `return_record` param if we wish.